lib/src/rpc/dial.dart

```diff
@@ -99,10 +104,28 @@ class DialWebRtcOptions {
   String? signalingAccessToken;
 }
 
+String allowedAddress = '';
```

Comment:
What is `allowedAddress` for, and why is it global?

Reply:
left over from a previous attempted implementation. Removing :)